### DISCUSSION TITLE
Fediverse?

### DISCUSSION THREAD

**Discussion by rcoder** [calagator/calagator#713] [Ideas]
Hello! It's been a while since I had anything to do with hacking on calagator -- about [15 years](https://github.com/rcoder/pdxruby/commit/b328f80f2dab902333a5a41df19e75880d5a3b42), in fact, predating this name/domain/maintainer list! -- but I've been thinking about the project + community a fair bit recently as we've all been watching the Fediverse thing finally, maybe, happen for real this time.

I'm not sure who actually has hands on the wheel for calagator these days, but if anyone wanted to brainstorm and/or start some experiments on how to bring the project into the Fediverse era, or provide a migration path for folks who want to run e.g. [gath.io](https://gath.io) or [Mobilizon](https://mobilizon.org/en/) I'd love to talk about it. (If you're in PDX, we could even have the discussion at the Lucky Lab, just for maximum old-skool PDX.rb flavor.)

**Comment by turoczy** [comment]
The last back and forth I had on Calagator stuff (regarding spam on the Portland instance) was with @reidab, so he may have thoughts.
